icu4c/source/common/uresdata.cpp

```diff
@@ -1040,13 +1040,15 @@ enum {
 /* The table item key string is not locally available. */
 static const char *const gUnknownKey="";
 
+#if !UCONFIG_NO_COLLATION
 /* resource table key for collation binaries: "%%CollationBin" */
 static const char16_t gCollationBinKey[]={
     0x25, 0x25,
     0x43, 0x6f, 0x6c, 0x6c, 0x61, 0x74, 0x69, 0x6f, 0x6e,
     0x42, 0x69, 0x6e,
     0
 };
```

Comment:
optional:
```suggestion
// resource table key for collation binaries
static const char16_t gCollationBinKey[]=u"%%CollationBin";
```

Reply:
ooh - i didn't look at the data close enough I guess. Good suggestion.